modules/mono/utils/path_utils.cpp

```diff
@@ -62,9 +62,7 @@ String cwd() {
 	}
 
 	String result;
-	if (result.parse_utf16(buffer.ptr())) {
-		return ".";
-	}
+	result.parse_utf16(buffer.ptr());
```

Comment:
This will now return an empty string I suppose, while it seems this is expected to return `"."` on failure.

Reply:
Added an extra check for the empty string.